### PR TITLE
Add $ENCS

### PR DESCRIPTION
**Pull request by encryptedcoins**
Please, add $ENCS token. PolicyID is in the pinned tweet at https://twitter.com/ENCOINS1. Thanks!

### CODE REVIEW

File: tokens.json [hunk 1]
```diff
@@ -1153,5 +1153,15 @@
       "twitter": "https://twitter.com/KryptoThreadz",
       "discord": "https://discord.gg/2Fs4ERC9Re"
     }
+  },
+  "9abf0afd2f236a19f2842d502d0450cbcd9c79f123a9708f96fd9b96": {
+    "project": "ENCOINS $ENCS",
+    "categories": ["DeFi", "DAO", "Other"]
```
Comment (theuranus) [inline tokens.json:1159]: Your PR is failed CI. You missed a comma here

Comment (encryptedcoins) [reply tokens.json]: Thanks, fixed.

File: tokens.json [hunk 2]
```diff
@@ -1153,5 +1153,15 @@
       "twitter": "https://twitter.com/KryptoThreadz",
       "discord": "https://discord.gg/2Fs4ERC9Re"
     }
+  },
+  "9abf0afd2f236a19f2842d502d0450cbcd9c79f123a9708f96fd9b96": {
+    "project": "ENCOINS $ENCS",
+    "categories": ["DeFi", "DAO", "Other"],
+    "socialLinks"{
```
Comment (theuranus) [inline tokens.json:1160]: and a colon here @encryptedcoins

File: tokens.json [hunk 3]
```diff
@@ -1153,5 +1153,15 @@
...
       "discord": "https://discord.gg/2Fs4ERC9Re"
     }
+  },
+  "9abf0afd2f236a19f2842d502d0450cbcd9c79f123a9708f96fd9b96": {
+    "project": "ENCOINS $ENCS",
+    "categories": ["DeFi", "DAO", "Other"],
+    "socialLinks": {
+      "website": "https://encoins.io",
+      "twitter": "https://twitter.com/ENCOINS1",
+      "discord": "discord.gg/Q3gPP87Tcw",
```
Comment (theuranus) [inline tokens.json:1163]: your discord link is not valid. 
Have to start with https

**Review by theuranus — COMMENTED** [review]
You can run 'npm run test' before push changes

**Comment by theuranus** [comment]
@tvtduy59 please help me verify this token tmr.

**Comment by tvtduy59** [comment]
LGTM @theuranus

**Review by tvtduy59 — APPROVED** [review]

**Pull request merged**